src/ia-book-visual-adjustments.js

```diff
@@ -0,0 +1,86 @@
+import { html, LitElement } from 'lit-element';
+import bookVisualAdjustmentsCSS from './styles/ia-book-visual-adjustments.js';
+import closeIcon from './icon_close.js';
+
+export class IABookVisualAdjustments extends LitElement {
+  static get styles() {
+    return bookVisualAdjustmentsCSS;
+  }
+
+  static get properties() {
+    return {
+      options: { type: Array },
+    };
+  }
+
+  constructor() {
+    super();
+
+    this.options = [];
+  }
+
+  get activeOptions() {
+    return this.options.reduce((results, option) => (
+      option.active ? [...results, option.id] : results
+    ), []);
+  }
+
+  get activeAdjustments() {
+    const count = this.activeOptions.length;
+    return count ? html`<p>(${count} active)</p>` : html``;
+  }
+
+  get adjustmentCheckboxes() {
+    return this.options.map((option) => {
+      const formID = `adjustment_${option.id}`;
+      return html`<li>
+        <label for="${formID}">
+          <span class="name">${option.name}</span>
+          <input type="checkbox" name="${formID}" id="${formID}" @change=${() => this.changeActiveStateFor(option.id)} ?checked=${option.active} />
+          <span class="icon"></span>
+        </label>
+        <p>${option.description}</p>
```

Comment:
_Suggestion_ Turn the form options into a child component. There's a decent amount of markup and styling that can isolated inside the component.